test/unit_tests/unit_test_image_function.cpp

```diff
@@ -1935,8 +1935,8 @@ namespace Function_Template
     {
         const std::vector < uint8_t > intensity = intensityArray( 2 );
         PenguinV_Image::Image image  = uniformImage( intensity[0] );
-        const uint32_t x = randomValue<uint32_t>( 0, image.width() );
-        const uint32_t y = randomValue<uint32_t>( 0, image.height() );
+        const uint32_t x = randomValue<uint32_t>( 0, image.width() - 1 );
```

Comment:
I'm not sure that this format is good as another developer would definitely forget to add `- 1`. What's your opinion?